### PR TITLE
Provide a note stating that this task is included in the Dropwizard core

### PR DESCRIPTION
**Pull request by bripkens**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [![Build Status](https://travis-ci.org/stuartgunter/dropwizard-logging-config.png?branch=master)](https://travis-ci.org/stuartgunter/dropwizard-logging-config)
 
+**Note: This task is now included in the Dropwizard core and is enabled by default for the admin environment.**
```
Comment (stuartgunter) [inline README.md]: Thanks for adding the note. Please could you include the version of Dropwizard that includes this task so anyone looking at it knows when the library is needed or not?

**Comment by bripkens** [comment]
@stuartgunter: Done, see https://github.com/dropwizard/dropwizard/commit/2f9ec0ee66c35ca9f7be32505278fe320b24faf6

**Comment by bripkens** [comment]
This is the earliest reference that I could fine in the repository.

**Comment by bripkens** [comment]
or rather this commit by you: https://github.com/dropwizard/dropwizard/commit/8eecb0aa5c74be0418bb3d9fb2fd85ff92e860e3

:)

**Pull request merged**